Common/Orders/MarketOrder.cs

```diff
@@ -42,6 +42,50 @@ public override decimal Value
         }
 
         /// <summary>
+        /// Create update request for pending orders. Null values will be ignored.
+        /// </summary>
+        public UpdateOrderRequest UpdateRequest(int? quantity = null, string tag = null)
```

Comment:
Please rename this and the other `UpdateRequest` functions to `CreateUpdateRequest`